services/thermal_mgr/thermal_mgr.c

```diff
@@ -38,23 +40,43 @@ void initThermalSystemManager(lm75bd_config_t *config) {
   thermalMgrQueueHandle = xQueueCreateStatic(
     THERMAL_MGR_QUEUE_LENGTH, THERMAL_MGR_QUEUE_ITEM_SIZE,
     thermalMgrQueueStorageArea, &thermalMgrQueueBuffer);
-
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY);
```

Comment:
Add error handling for this:
- check that the event is not null, return invalid argument otherwise
- check that the thermalMgrQueueHandle is not null, otherwise return invalid state
- check the return value of the xQueueSend

Also this should be 0 and not PORTMAX_DELAY as this will block the task which might not be the intended behavior especially in ISR